src/generator/AutoRest.Ruby/Model/MethodRb.cs

```diff
@@ -606,6 +606,16 @@ public string GetDeserializationString(IModelType type, string valueReference =
                 builder.AppendLine("{1} = @client.deserialize(result_mapper, {0}, '{1}')", responseVariable, valueReference);
             }
 
+            if (type is CompositeType)
```

Comment:
would this added code be only applicable to Azure related methods? If so, should this be part of AutoRest.Ruby.Azure, instead of AutoRest.Ruby?